kshebeko/lesson4/src/com/company/kshebeko_class/Flower.java

```diff
@@ -0,0 +1,27 @@
+package com.company.kshebeko_class;
+
+public abstract class Flower {
+    protected String name;
+    protected int price;
+
+    public Flower(String name, int price) { //Конструктор
+        this.name = name;
+        this.price = price;
+    }
+
+    public Flower() { //Конструктор
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String toString() {
+        return "Flower name: " + name + " Price  = " + price;
+    }
+
+    protected abstract String setName();
```

Comment:
setters are usually void as they set value in class field, they don't return any values

Reply:
still not fixed